doc/sphinx/using/libraries/writing.rst

```diff
@@ -37,11 +37,21 @@ deprecated compatibility alias using :cmd:`Notation (abbreviation)`
              Ltac2 definition @qualid is deprecated since @string__since. @string__note
              Ltac2 alias @qualid is deprecated since @string__since. @string__note
              Ltac2 notation {+ @ltac2_scope } is deprecated since @string__since. @string__note
+             Library File @qualid is deprecated since @string__since. @string__note
+             Library File (transitively required) @qualid is deprecated since @string__since. @string__note
```

Comment:
Maybe this would be better in alphabetic order?  The rationale behind the current order is unclear.